WinUIGallery/ControlPages/TitleBarPage.xaml.cs

```diff
@@ -128,5 +165,48 @@ private void defaultTitleBar_Click(object sender, RoutedEventArgs e)
             // announce visual change to automation
             UIHelper.AnnounceActionForAccessibility(sender as UIElement, "TitleBar size and width changed", "TitleBarChangedNotificationActivityId");
         }
+
+        private void AddInteractiveElements_Click(object sender, RoutedEventArgs e)
+        {
+            var txtBoxNonClientArea = UIHelper.FindElementByName(sender as UIElement, "AppTitleBarTextBox") as FrameworkElement;
+
+            if (txtBoxNonClientArea.Visibility == Visibility.Visible)
+            {
+                ResetTitlebarSettings();
+            }
+            else
+            {
+                addInteractiveElements.Content = "Remove interactive control from titlebar";
+                txtBoxNonClientArea.Visibility = Visibility.Visible;
+                if (!sizeChangedEventHandlerAdded)
+                {
+                    sizeChangedEventHandlerAdded = true;
+                    // run this code when textbox has been made visible and its actual width and height has been calculated
+                    txtBoxNonClientArea.SizeChanged += (object sender, SizeChangedEventArgs e) =>
+                    {
+                        if (txtBoxNonClientArea.Visibility != Visibility.Collapsed)
+                        {
+                            GeneralTransform transformTxtBox = txtBoxNonClientArea.TransformToVisual(null);
+                            Rect bounds = transformTxtBox.TransformBounds(new Rect(0, 0, txtBoxNonClientArea.ActualWidth, txtBoxNonClientArea.ActualHeight));
+
+                            var scale = WindowHelper.GetRasterizationScaleForElement(this);
+
+                            var transparentRect = new Windows.Graphics.RectInt32(
+                                _X: (int)Math.Round(bounds.X * scale),
+                                _Y: (int)Math.Round(bounds.Y * scale),
+                                _Width: (int)Math.Round(bounds.Width * scale),
+                                _Height: (int)Math.Round(bounds.Height * scale)
+                            );
+                            var rectArr = new Windows.Graphics.RectInt32[] { transparentRect };
+                            SetClickThruRegions(rectArr);
+                        }
+                    };
```

Comment:
Is this the recommended logic for setting the region rects? If so, I think this should be part of TitleBarSample3.txt so people can see the code without needing to dig into the gallery repo.

Reply:
it won't be good to have this included in the sample. it is too big, too specific and would confuse the users more.  the point of `TitleBarSample3.txt ` is how users can use `SetRegionRects` to create interactive regions within titlebar.
How will they get the coordinates for rect ?
They can manually define the coordinates, get it using this method or something else. Hence, I am keeping the example generic. I have already mentioned in the sample to look at this implementation if they want to see "a way" of doing it.

Reply:
How are apps going to calculate their rects in a different way? I worry without sharing how to compute the rects (including properly handling the scale factor) that this sample will be too incomplete to be useful.

Reply:
though i think it will complicate the sample but I see your point. We can avoid a whole class of issues where people are passing wrong coordinates and their drag regions not working. In interest of that, I have decided to add all the info in the sample.